test/api/aphw-ddi-api.postman_collection.json

```diff
@@ -1922,6 +1922,162 @@
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{DDI_API_BASE_URL}}/cdo",
+									"host": [
+										"{{DDI_API_BASE_URL}}"
+									],
+									"path": [
+										"cdo"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "/dog invalid",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"response returns 404 given invalid dog index number requested\", function () {",
+											"    pm.response.to.have.status(404);",
+											"});"
+										],
+										"type": "text/javascript",
+										"packages": {}
+									}
+								}
+							],
+							"request": {
+								"method": "DELETE",
+								"header": [
+									{
+										"key": "ddi-username",
+										"value": "internal-user",
+										"type": "text"
+									},
+									{
+										"key": "ddi-displayname",
+										"value": "User, Internal",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{DDI_API_BASE_URL}}/dog/ED9999999999",
+									"host": [
+										"{{DDI_API_BASE_URL}}"
+									],
+									"path": [
+										"dog",
+										"ED9999999999"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "/dog",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"response returns a 204 and empty body given\", function () {",
+											"    pm.response.to.have.status(204);",
+											"    const body = pm.response.body",
+											"    pm.expect(body).to.eq(undefined)",
+											"});"
+										],
+										"type": "text/javascript",
+										"packages": {}
+									}
+								}
+							],
+							"request": {
+								"method": "DELETE",
+								"header": [
+									{
+										"key": "ddi-username",
+										"value": "internal-user",
+										"type": "text"
+									},
+									{
+										"key": "ddi-displayname",
+										"value": "User, Internal",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{DDI_API_BASE_URL}}/dog/{{dogIndexToDelete}}",
+									"host": [
+										"{{DDI_API_BASE_URL}}"
+									],
+									"path": [
+										"dog",
+										"{{dogIndexToDelete}}"
+									]
+								}
+							},
+							"response": []
+						}
+					]
+				}
+			]
+		},
```

Comment:
Did I miss a test for the actual successful delete of a dog? Also, I notice you only add one dog in the CDO. It might be worth the CDO adding 2 dogs so you can successfully delete one without hitting the 'delete last dog of an owner' scenario